src/analyzer/protocol/mime/MIME.cc

```diff
@@ -1131,7 +1131,7 @@ void MIME_Entity::DecodeBase64(int len, const char* data)
 		{
 		rlen = 128;
 		char* prbuf = rbuf;
-		int decoded = base64_decoder->Decode(len, data, &rlen, &prbuf);
+		int decoded = base64_decoder->Decode(len, data, reinterpret_cast<uint64_t*>(&rlen), &prbuf); // typecasting for now
```

Comment:
This does not look right to me.

This takes a pointer to an int (length can be 2 or 4 bytes, depending on system), and casts it to an uint64_t (which has a length of 4 bytes). In cases where sizeof(int) != sizeof(uint64_t), that reads into the following memory space, right?

Also - this should probably be size_t too now (which does not fix this problem).